aleph-client/src/multisig.rs

```diff
@@ -122,6 +125,7 @@ impl SignatureAggregation {
 
 /// `MultisigParty` is representing a multiparty entity constructed from
 /// a group of accounts (`members`) and a threshold (`threshold`).
+#[derive(Clone)]
```

Comment:
[question] cannot we make it `(Partial)Eq`?

Reply:
`KeyPair` is not `Eq`, so not just like that.